helper-cli/src/main/kotlin/commands/packageconfig/CreateCommand.kt

```diff
@@ -174,8 +183,15 @@ internal class CreateCommand : CliktCommand(
         }
 
     private fun getNonOffendingLicenses(): Set<SpdxSingleLicenseExpression> {
+        val result = mutableSetOf<SpdxSingleLicenseExpression>()
```

Comment:
To me `result` indicates that this will be the return value of the function. I believe it's quite clear and also widely used in ORT. I'd prefer to keep it.